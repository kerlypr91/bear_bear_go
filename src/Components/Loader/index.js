import React from 'react'

import './style.scss'

export default function Loader () {
  return (
    <div className="component">
      <div className="component__loader" />
    </div>
  )
}