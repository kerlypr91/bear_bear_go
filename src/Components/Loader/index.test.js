import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './index';

test('Loader match snapshot', () => {
  const component = renderer.create(
    <Loader />
  )

  // base snapshot

  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()
});