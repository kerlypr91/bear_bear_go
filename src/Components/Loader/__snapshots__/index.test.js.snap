// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader match snapshot 1`] = `
<div
  className="component"
>
  <div
    className="component__loader"
  />
</div>
`;
